apps/web/tests/components/submission/components/Credential.test.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Credential } from '../../../../src/components/submission/components/Credential';
+import {
+  defaultSpecialtyValue,
+  registrationStatusOptions,
+  healthAuthorityOptions,
+  employmentOptions,
+} from '../../../../src/components/submission/validation/credential';
```

Comment:
Total nit, but we might want to spend a bit of time figuring out pathing for the project.

Reply:
Jest didn't like the TS absolute pathing, if we are able to fix that configuration it'll look a lot better